rules/crowdstrike_rules/crowdstrike_connection_to_embargoed_country.py

```diff
@@ -0,0 +1,41 @@
+from panther_base_helpers import crowdstrike_network_detection_alert_context, deep_get
+
+# U.S. Gov Sanctioned Destinations
+EMBARGO_COUNTRY_CODES = {
+    "CU",  # Cuba
+    "IR",  # Iran
+    "KP",  # DPRK
+    "SY",  # Syria
+}
+
+
+def get_enrichment_obj(event):
+    return deep_get(event, "p_enrichment", "ipinfo_location", "p_any_ip_addresses", default=None)
+
+
+def rule(event):
+    enrichment_obj = get_enrichment_obj(event)
+    # enrichment_object returns a list.
+    # Iterate over list and check if the "country" value matches the country codes.
+    if enrichment_obj:
+        if [i for i in enrichment_obj if i["country"] in EMBARGO_COUNTRY_CODES]:
+            return True
```

Comment:
more personal pref but easier to understand as:
```suggestion
        for i in enrichment_obj:
          if i.get('country') in EMBARGO_COUNTRY_CODES:
            return True
```
negligible compute savings as well